Packages/ClientRuntime/Sources/Networking/SdkError.swift

```diff
@@ -14,3 +14,18 @@ public enum SdkError<E>: Error {
   case unknown(Error?)
 
 }
+
+
+extension SdkError: CodedError {
+
+    public var errorCode: String? {
```

Comment:
Can we add docs as to what an error code is (or should be) for a SdkError?

Reply:
Added